ci/nightly-test.sh

```diff
@@ -106,6 +106,13 @@ start_run_teardown "itst02.sh"
 start_run_teardown "itst11.sh"
 start_run_teardown "itst13.sh"
 start_run_teardown "itst14.sh"
+start_run_teardown "itst01_private_chain.sh"
+start_run_teardown "itst02_private_chain.sh"
+start_run_teardown "itst06_private_chain.sh"
+start_run_teardown "itst07_private_chain.sh"
+start_run_teardown "itst11_private_chain.sh"
+# start_run_teardown "itst13_private_chain.sh"
+# start_run_teardown "itst14_private_chain.sh"
```

Comment:
Can these be enabled?

Reply:
Ticketed fix to these scripts so they're safe to be enabled